quartz/styles/base.scss

```diff
@@ -30,6 +30,10 @@ section {
   color: var(--darkgray);
 }
 
+svg {
```

Comment:
can we make this not apply to every `svg` (i.e. let's get a more specific selector in here)

Reply:
actually I forgot to remove this. Since this would override the SVG that is used in the icon 😄

Reply:
done now.